serialize/serialize.go

```diff
@@ -55,17 +54,26 @@ func encode(w io.Writer, value interface{}) error {
 }
 
 // Load reads given file and returns the read config, or error.
-func Load(filename string) (*config.Config, error) {
+func Load(filename string, cfg interface{}) error {
```

Comment:
Where's the cyclic import this is fixing?

Reply:
It makes use of code that lives/lived one directory up, so `gx-go rw` wasn't able to rewrite the import path here

Reply:
Ah, this PR branch is based off #9

Reply:
Hm. I can't reproduce. Let's discuss there.